tests/config/nominal/.wakeroot

```diff
@@ -1,3 +1,5 @@
 {
-  "user_config": "./wake_user.json"
+  "user_config": "./wake_user.json",
+  "log_header":"",
```

Comment:
I still want the `nominal` case to set these values to something arbitrary.

ie 
```json
{
  "user_config": "./wake_user.json",
  "log_header": "$label",
  "log_header_source_width": 13
}
```